pom.xml

```diff
@@ -106,8 +106,28 @@
                     <imagesDir>${basedir}/docs/modules/ROOT/images</imagesDir>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
This is in the main pom.xml. Do we need to add plugin here? just a question.

Reply:
I removed them.